learn/installing-ballerina.md

```diff
@@ -52,13 +49,13 @@ The installer should put the `C:\Program Files\Ballerina\<ballerina-directory>\b
 
 ### Installing on Linux
 
-* [Download](/downloads) the latest version of Ballerina.
+[Download the DEB file](/downloads) and double-click on it to launch the installer. The installer guides you through the installation process and installs the Ballerina distribution to `/usr/lib/ballerina`.
 
-* If you downloaded the DEB file, use the following command to install Ballerina. Replace `<ballerina-home>/<ballerina-binary>.deb` with the actual file path. This installs Ballerina to the `/usr/lib/ballerina` directory.
+> **Info:** Alternatively, you can use the below command to install Ballerina using the downloaded the DEB file. Replace `<ballerina-home>/<ballerina-binary>.deb` with the actual file path.
 
-  ```
-  dpkg -i <ballerina-binary>.deb
-  ```
+```
+dpkg -i <ballerina-binary>.deb
```

Comment:
@praneesha Shall we include rpm installation as well?

Reply:
@tharik - What's the command for the RPM installation? Also, we haven't given an RPM installation for the latest release.

Reply:
Fixed the above.